app-web/src/components/Card/CardsInColumns.js

```diff
@@ -35,7 +35,7 @@ const CardsInColumns = ({ cards }) => {
         title={card.fields.title}
         description={card.fields.description}
         image={card.fields.image}
-        link={card.fields.standAlonePath}
+        link={card.fields.standAlonePath || getFirstNonExternalResource(card.connectsWith)}
```

Comment:
we  want to move this implementation outside of the comopnent that calls  it. This is better places  at the index.js level